DAQ_System/Source/main.cpp

```diff
@@ -112,117 +103,117 @@ int main()
         fflush(stdout);
 
         // Get current stream position
-        long    pos = ftell(f);
+        long pos = ftell(numbers_file);
 
         // Parse out the number and increment
         int32_t number;
-        fscanf(f, "%ld", &number);
+        fscanf(numbers_file, "%ld", &number);
         number += 1;
 
         // Seek to beginning of number
-        fseek(f, pos, SEEK_SET);
+        fseek(numbers_file, pos, SEEK_SET);
 
         // Store number
-        fprintf(f, "    %ld\n", number);
+        fprintf(numbers_file, "    %ld\n", number);
 
         // Flush between write and read on same file
-        fflush(f);
+        fflush(numbers_file);
     }
 
     printf("\rIncrementing numbers (%d/%d)... OK\n", 10, 10);
 
     // Close the file which also flushes any cached writes
     printf("Closing \"/fs/numbers.txt\"... ");
     fflush(stdout);
-    err = fclose(f);
-    printf("%s\n", (err < 0 ? "Fail :(" : "OK"));
-    if (err < 0) {
+    status = fclose(numbers_file);
+    printf("%s\n", (status < 0 ? "Fail :(" : "OK"));
+    if (status < 0) {
         error("error: %s (%d)\n", strerror(errno), -errno);
     }
 
     // Display the root directory
     printf("Opening the root directory... ");
     fflush(stdout);
 
-    DIR*    d = opendir("/fs/");
-    printf("%s\n", (!d ? "Fail :(" : "OK"));
-    if (!d) {
+    DIR* root_dir = opendir("/fs/");
+    printf("%s\n", (!root_dir ? "Fail :(" : "OK"));
+    if (!root_dir) {
         error("error: %s (%d)\n", strerror(errno), -errno);
     }
 
     printf("root directory:\n");
     while (true) {
-        struct dirent*  e = readdir(d);
-        if (!e) {
+        struct dirent* entry = readdir(root_dir);
+        if (!entry) {
             break;
         }
 
-        printf("    %s\n", e->d_name);
+        printf("    %s\n", entry->d_name);
     }
 
     printf("Closing the root directory... ");
     fflush(stdout);
-    err = closedir(d);
-    printf("%s\n", (err < 0 ? "Fail :(" : "OK"));
-    if (err < 0) {
+    status = closedir(root_dir);
+    printf("%s\n", (status < 0 ? "Fail :(" : "OK"));
+    if (status < 0) {
         error("error: %s (%d)\n", strerror(errno), -errno);
     }
```

Comment:
Thoughts on whether we should support this? This may be outside of our focus, as it may concern the use of a file system instead of logging to the sd card 🤔